lib/adapters/default.js

```diff
@@ -0,0 +1,34 @@
+const { StorageManager } = require('brainstem-js');
+
+module.exports = {
+  fetchCollection: (brainstemKey, options) => {
+    const storageManager = StorageManager.get();
+    return storageManager.storage(brainstemKey)
+      .fetch(Object.assign(options.fetchOptions, { remove: false }));
```

Comment:
We should never be modifying anything we don't own -- unless you can guarantee that the object is thrown away.

Let's make sure we're cloning and creating a new object.

Reply:
Oh, I see that's how it was before. Let's still make the edit -- it happens that the previous code was bad.